Assets/Samples/SimpleDemo/SimpleControls.cs

```diff
@@ -50,7 +59,18 @@ public @SimpleControls()
                     ""path"": ""*/{PrimaryAction}"",
                     ""interactions"": """",
                     ""processors"": """",
-                    ""groups"": """",
+                    ""groups"": ""Gamepad;KeyboardMouse"",
```

Comment:
Ooppps... any of the changes from here to the end of the file are accidental commits. Will revert.